x-pack/plugins/canvas/canvas_plugin_src/renderers/reveal_image/index.tsx

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { RendererStrings } from '../../../i18n';
+import { RendererFactory } from '../../../types';
+import { Output as Arguments } from '../../functions/common/revealImage';
+import { RevealImage } from './components/reveal_image';
+
+const { revealImage: strings } = RendererStrings;
+
+export const revealImage: RendererFactory<Arguments> = () => ({
+  name: 'revealImage',
+  displayName: strings.getDisplayName(),
+  help: strings.getHelpDescription(),
+  reuseDomNode: false,
+  render(domNode, config, handlers) {
+    // modify the top-level container class
+    domNode.className = 'revealImage';
+
+    ReactDOM.render(
+      <RevealImage
+        image={config.image}
+        emptyImage={config.emptyImage}
+        percent={config.percent}
+        origin={config.origin}
+        width={domNode.clientWidth}
+        height={domNode.clientHeight}
+      />,
+      domNode,
+      () => handlers.done()
+    );
+
+    handlers.onResize(() => {
```

Comment:
@crob611 How do we update this for the new handlers?

Reply:
Should be able to to `handlers.event('onResize', () => { ....`

All the "event" handlers are defined here https://github.com/elastic/kibana/blob/master/x-pack/plugins/canvas/public/lib/create_handlers.ts#L98